specification/redis/resource-manager/Microsoft.Cache/stable/2023-04-01/redis.json

```diff
@@ -2042,6 +2042,17 @@
         },
         "subnetId": {
           "type": "string",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.Network/virtualNetworks/subnets"
+              },
+              {
+                "type": "Microsoft.ClassicNetwork/virtualNetworks/subnets"
```

Comment:
Is ClassicNetwork still supported? I think the plan is to deprecate it soon.

Reply:
I'm not sure, but I don't think it hurts to have it here as all this is really saying is that "we would accept ClassicNetwork if you give it to us" - even if you can't make new ones maybe legacy existing ones are still allowed for a while?

Agree that it might be nice to be able to remove this once it really is deprecated, though not sure how to actually accomplish that in practice when the time comes.